stubs/inertia/resources/js/Pages/OAuth/Partials/OAuthAppManager.vue

```diff
@@ -0,0 +1,318 @@
+<script setup>
+import { ref } from 'vue';
+import { useForm } from '@inertiajs/vue3';
+import ActionMessage from '@/Components/ActionMessage.vue';
+import ActionSection from '@/Components/ActionSection.vue';
+import Checkbox from '@/Components/Checkbox.vue';
+import ConfirmationModal from '@/Components/ConfirmationModal.vue';
+import DangerButton from '@/Components/DangerButton.vue';
+import DialogModal from '@/Components/DialogModal.vue';
+import FormSection from '@/Components/FormSection.vue';
+import InputError from '@/Components/InputError.vue';
+import InputLabel from '@/Components/InputLabel.vue';
+import PrimaryButton from '@/Components/PrimaryButton.vue';
+import SecondaryButton from '@/Components/SecondaryButton.vue';
+import SectionBorder from '@/Components/SectionBorder.vue';
+import TextInput from '@/Components/TextInput.vue';
+
+const props = defineProps({
+    apps: Array,
+});
+
+const createOAuthAppForm = useForm({
+    name: '',
+    redirect_uris: [],
+    confidential: false,
+});
+
+const updateOAuthAppForm = useForm({
+    name: '',
+    redirect_uris: [],
+});
+
+const deleteOAuthAppForm = useForm({});
+
+const displayingClientCredentials = ref(false);
+const oauthAppBeingManaged = ref(null);
+const oauthAppBeingDeleted = ref(null);
+
+const createOAuthApp = () => {
+    createOAuthAppForm.post(route('oauth-apps.store'), {
+        preserveScroll: true,
+        onSuccess: () => {
+            displayingClientCredentials.value = true;
+            createOAuthAppForm.reset();
+        },
+    });
+};
+
+const manageOAuthApp = (app) => {
+    updateOAuthAppForm.name = app.name;
+    updateOAuthAppForm.redirect_uris = app.redirect.split(',');
```

Comment:
`app.redirect` is undefined. `app.redirect_uris` is an array.

Reply:
Thanks for your review. I was waiting for laravel/passport#1744 to be merged. Fixed now.

Reply:
Yeah this one is indented, Didn't want to make the view too complex to support multiple redirect URIs. The view is customizable, so one can add a button for adding more fields dynamically if wanted.